velox/functions/prestosql/tests/RepeatTest.cpp

```diff
@@ -124,3 +124,28 @@ TEST_F(RepeatTest, repeatWithInvalidCount) {
       {elementVector},
       "(10001 vs. 10000) Count argument of repeat function must be less than or equal to 10000");
 }
+
+TEST_F(RepeatTest, repeatAllowNegativeCount) {
```

Comment:
Move this to lib folder.

Reply:
Assume we need to move the entire test file to lib folder. Just updated. Thanks!